File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/parking/GetParkingFeeForFleetDataResponse.java
package com.lattis.ellipse.data.network.model.response.parking;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 8/1/17.
 */

public class GetParkingFeeForFleetDataResponse {

    @SerializedName("outside")
    private boolean outside;
    @SerializedName("not_allowed")
    private boolean not_allowed;
    @SerializedName("fee")
    private float fee;
    @SerializedName("currency")
    private String currency;

    public String getCurrency() {
        return currency;
    }

    public boolean isOutside() {
        return outside;
    }

    public boolean isNot_allowed() {
        return not_allowed;
    }

    public float getFee() {
        return fee;
    }


}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/parking/GetParkingFeeForFleetDataResponse.kt
package com.lattis.data.entity.response.parking

import com.google.gson.annotations.SerializedName

class GetParkingFeeForFleetDataResponse {
    @SerializedName("outside")
    val isOutside = false

    @SerializedName("not_allowed")
    val isNot_allowed = false

    @SerializedName("fee")
    val fee = 0f

    @SerializedName("currency")
    val currency: String? = null

}